bootstrap-joining-demo/kubernetes-api/src/main/resources/application.conf

```diff
@@ -6,8 +6,13 @@ akka {
 
   discovery {
     # pick the discovery method you'd like to use:
-//    method = akka-dns
+    # method = akka-dns
     method = kubernetes-api
+
+    kubernetes-api {
+      pod-label-selector = "app=appka"
```

Comment:
For this example, I think we should add labels for `actorSystemName` and use that. So, `pod-label-selector=actorSystemName=%s`, and add a label to the pod `actorSystemName: appka`.

Also, would you mind removing line 9 (the commented out akka-dns)?